taxonomy-manager-engine/src/main/java/com/digirati/taxman/analysis/WordTokenSearchStrategy.java

```diff
@@ -13,4 +13,6 @@
     void unindex(WordTokenSearchEntry<IdT> entry);
 
     Set<IdT> match(List<WordToken> input);
+
+    Set<IdWithPosition<IdT>> matchWithPosition(List<WordToken> input);
```

Comment:
Let's get rid of the default `match(...)` and rename `matchWithPosition(...)` to `match(...)`, making it the default.